test/runtime/lb.go

```diff
@@ -301,6 +301,103 @@ var _ = Describe("RuntimeLB", func() {
 		status = vm.ContainerExec("client", testCmd)
 		status.ExpectSuccess("LB is not working after restart")
 	})
+
+	Context("Services Policies", func() {
+		BeforeEach(func() {
+			vm.PolicyDelAll()
+		})
+
+		AfterEach(func() {
+			vm.SampleContainersActions(helpers.Delete, helpers.CiliumDockerNetwork)
+			vm.PolicyDelAll()
```

Comment:
Why is the `*CmdRes` from `PolicyDelAll` being ignored here? We should either explicitly ignore the output of functions using `_` and add a comment explaining why we are ignoring the return value(s) of a function or check the return values for specific conditions. Silently ignoring output might make debugging of tests more difficult.